medical/gql_mutations.py

```diff
@@ -91,7 +80,11 @@ class ItemOrServiceInputType(OpenIMISMutation.Input):
 
 class ServiceInputType(ItemOrServiceInputType):
     level = graphene.String(required=True)
+    packagetype = graphene.String(required=True)
```

Comment:
Having True create  regression issue, I would rather have a default value

```suggestion
    packagetype = graphene.String(required=False)
```